draft-ietf-tls-dtls13.md

```diff
@@ -347,42 +347,27 @@ allowing the sequence number to wrap.
 Implementations MUST NOT allow the epoch to wrap, but instead MUST establish
 a new association, terminating the old association.
 
-
 ##  Transport Layer Mapping
 
-Each DTLS record MUST fit within a single datagram.  In order to
-avoid IP fragmentation, clients of the DTLS record layer SHOULD
-attempt to size records so that they fit within any PMTU estimates
-obtained from the record layer.
+### Fragmentation and Reassembly 
 
-Note that unlike IPsec, DTLS records do not contain any association
-identifiers.  Applications must arrange to multiplex between associations.
-With UDP, the host/port number is used to look up the appropriate
-security association for incoming records.
+When a DTLS library transmits handshake messages and application data
+it has to decide how much data to place in a single DTLS record and
+how many records to encapsulate within a single datagram. One one 
+hand the sender SHOULD attempt to size records so that they fit within 
+any PMTU estimates obtained from the record layer and on the other 
+hand it needs to take the memory limitations expressed via the 
+Maximum Fragment Length extension {{RFC6066}} or the Record Size 
+Limit extension {{I-D.ietf-tls-record-limit}} into account. 
 
-Multiple DTLS records may be placed in a single datagram.  They are
-simply encoded consecutively.  The DTLS record framing is sufficient
-to determine the boundaries.  Note, however, that the first byte of
-the datagram payload must be the beginning of a record.  Records may
-not span datagrams.
+There are, however, rules an implementation needs to adhere to:
 
-Some transports, such as DCCP {{RFC4340}}, provide their own sequence
-numbers.  When carried over those transports, both the DTLS and the
-transport sequence numbers will be present.  Although this introduces
-a small amount of inefficiency, the transport layer and DTLS sequence
-numbers serve different purposes; therefore, for conceptual simplicity,
-it is superior to use both sequence numbers.
+- Each DTLS record MUST fit within a single datagram. The first byte of
+the datagram payload must be the beginning of a record. 
 
-Some transports provide congestion control for traffic
-carried over them.  If the congestion window is sufficiently narrow,
-DTLS handshake retransmissions may be held rather than transmitted
-immediately, potentially leading to timeouts and spurious
-retransmission.  When DTLS is used over such transports, care should
-be taken not to overrun the likely congestion window. {{RFC5238}}
-defines a mapping of DTLS to DCCP that takes these issues into account.
-
-
-##  PMTU Issues
+- Multiple DTLS records may be placed in a single datagram. They are
+simply encoded consecutively. The DTLS record framing is sufficient
+to determine the boundaries.
```

Comment:
It might be worthwhile to reference the additional rules related to the record layer definition in Section 5 of draft-ietf-tls-tls13.

Reply:
Actually, I think this text is right. The first byte must be the beginning of a record but not vice versa.